### PR TITLE
Add oracledb

### PR DESCRIPTION
**Pull request by xylar**
This apparently replaces cx_oracle

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/oracledb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - LICENSE.txt
+    - NOTICE.txt
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - nicoddemus
```
Comment (xylar) [inline recipes/oracledb/meta.yaml:49]: @nicoddemus, it seems like you are the only active maintainer of https://github.com/conda-forge/cx_oracle-feedstock.  It seems that `oracledb` is just a rename (and new versioning) of `cx_oracle`.  Would you like to be a maintainer of the new package?

**Comment by xylar** [comment]
This package is needed for: https://github.com/conda-forge/apache-airflow-providers-oracle-feedstock/pull/10

**Comment by xylar** [comment]
@conda-forge/help-python-c, I could use some help understanding why I'm seeing this error with `clang`:
```
 x86_64-apple-darwin13.4.0-clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/runner/mambaforge/conda-bld/oracledb_1655972733475/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/include -fPIC -O2 -isystem /Users/runner/mambaforge/conda-bld/oracledb_1655972733475/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/include -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -isystem /Users/runner/mambaforge/conda-bld/oracledb_1655972733475/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/include -fdebug-prefix-map=/Users/runner/mambaforge/conda-bld/oracledb_1655972733475/work=/usr/local/src/conda/oracledb-1.0.1 -fdebug-prefix-map=/Users/runner/mambaforge/conda-bld/oracledb_1655972733475/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_=/usr/local/src/conda-prefix -D_FORTIFY_SOURCE=2 -isystem /Users/runner/mambaforge/conda-bld/oracledb_1655972733475/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/include -mmacosx-version-min=10.9 -I/Users/runner/mambaforge/conda-bld/oracledb_1655972733475/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/include/python3.8 -c src/oracledb/base_impl.c -o build/temp.macosx-10.9-x86_64-cpython-38/src/oracledb/base_impl.o -target arm64-apple-macos
  clang-13: error: the clang compiler does not support '-march=core2'
```
Is this something I need to deal with upstream or is it a conda-forge issue?  (I'm more of a python guy.)

**Comment by xylar** [comment]
@conda-forge/help-python-c, this is now ready to review.

**Review by chrisburr — APPROVED** [review]
Looks good, just needs a comment from @nicoddemus about if they'd like to be listed as a maintainer

File: recipes/oracledb/meta.yaml (hunk 1 above)
Comment (nicoddemus) [reply recipes/oracledb/meta.yaml:51]: Thanks xylar, but I will pass on this one: I already maintain a ton of packages and we stopped using cx-oracle. 😁

Comment (xylar) [reply recipes/oracledb/meta.yaml:51]: Okay, no problem.

**Comment by xylar** [comment]
Okay, @chrisburr, this is good to go and @nicoddemus has been removed at their request.

**Comment by xylar** [comment]
Thanks again, @chrisburr!

**Pull request merged**